src/main/resources/META-INF/resources/jsp/application/include/app-detail-elements/vacation_overview_js.jsp

```diff
@@ -1,4 +1,50 @@
 <script>
+
+	//https://tc39.github.io/ecma262/#sec-array.prototype.find
+	if (!Array.prototype.find) {
```

Comment:
wenn wir Polyfills in einer extra JavaScript Datei ablegen und diese immer einbinden, kann Array#find immer und überall verwendet werden ohne weitere Beachtung.

Reply:
Gute Idee!
Wo kann ich diese Datei denn zentral einbinden?
Eine index.html (außer src/main/resources/static/api/index.html) finde ich nicht.
Gib es eine andere zentrale JSP?

Reply:
das ist ein gute Frage :D kann der @honnel oder @christlang das beantworten? gibt es einen header der überall inkludiert wird?

Reply:
ja hier: https://github.com/synyx/urlaubsverwaltung/blob/master/src/main/resources/META-INF/resources/WEB-INF/tags/head.tag

Reply:
Danke, hab's gerade erledigt und gepushed.